config/deployments/README.md

```diff
@@ -0,0 +1,156 @@
+- scheduler
+  - consumer
+  - producer
+
+Hyperswitch maintains 3 different environments for deployments under the `router` component, namely,
+
+- Integration Test
+- Sandbox
+- Production
+
+To learn about what "router", "drainer" and "scheduler" is, please refer to the [Hyperswitch architecture][architecture] documentation.
+
+### Tree structure
+
+```text
+config/deployments      # Root directory for the deployment configs
+├── README.md           # This file
+├── drainer.toml        # Config specific to drainer
+├── env_specific.toml   # Config for environment specific values which are meant to be sensitive (to be set by the user)
+├── integ.toml          # Config specific to integration_test environment
+├── production.toml     # Config specific to production environment
+├── sandbox.toml        # Config specific to sandbox environment
+└── scheduler           # Directory for scheduler configs
+    ├── consumer.toml   # Config specific to consumer
+    └── producer.toml   # Config specific to producer
+```
+
+## Router
+
+The `integration_test.toml`, `sandbox.toml`, and `production.toml` files are configuration files for the environments `integration_test`, `sandbox`, and `production`, respectively. These files maintain a 1:1 mapping with the environment names, and it is recommended to use the same name for the environment throughout this document.
+
+### Generating a Config File for the Router
+
+The `env_specific.toml` file contains values that are specific to the environment. This file is kept separate because the values in it are sensitive and are meant to be set by the user. The `env_specific.toml` file is merged with the `integ.toml`, `sandbox.toml`, or `production.toml` file to create the final configuration file for the router.
+
+For example, to build and deploy Hyperswitch in the **sandbox environment**, you can duplicate the `env_specific.toml` file and rename it as `sandbox_config.toml`. Then, update the values in the file with the proper values for the sandbox environment.
+
+The environment-specific `sandbox.toml` file, which contains the Hyperswitch recommended defaults, is merged with the `sandbox_config.toml` file to create the final configuration file called `sandbox_release.toml`. This file is marked as ready for deploying on the sandbox environment.
+
+1. Duplicate the `env_specific.toml` file and rename it as `sandbox_config.toml`:
+
+```shell
+cp config/deployments/env_specific.toml config/deployments/sandbox_config.toml
+```
+
+2. Update the values in the `sandbox_config.toml` file with the proper values for the sandbox environment:
+
+```shell
+vi config/deployments/sandbox_config.toml
+```
+
+3. To merge the files you can use `cat`:
+
+```shell
+cat config/deployments/sandbox.toml config/deployments/sandbox_config.toml > config/deployments/sandbox_release.toml
+```
+
+> Note: You can refer to the [`config.example.toml`][config_example] file to understand the variables that used are in the `env_specific.toml` file.
```

Comment:
```suggestion
> [!NOTE]
> You can refer to the [`config.example.toml`][config_example] file to understand the variables that used are in the `env_specific.toml` file.
```

Optional change. This is an "alert" syntax in GitHub flavored Markdown, is rendered slightly better, more info: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts